includes/class-wc-stripe-api.php

```diff
@@ -227,34 +227,33 @@ public static function request( $request, $api = 'charges', $method = 'POST', $w
 		);
 
 		$response_headers = wp_remote_retrieve_headers( $response );
-		// Log the stripe version in the response headers, if present.
-		if ( isset( $response_headers['stripe-version'] ) ) {
```

Comment:
Just to confirm this removal was intentional (I imagine this information is not that useful anymore).

Reply:
Correct, this log entry is redundant; we always send the `stripe-version` in the headers (and was added to the webhook handling with the webhook autoconfiguration)